src/CMakeLists.txt

```diff
@@ -41,6 +42,12 @@ else()
 endif()
 target_link_libraries(bpftrace ${LIBELF_LIBRARIES})
 
+if (LIBBPF_FOUND AND NOT STATIC_LINKING)
```

Comment:
Is there no static library version of libbpf? I'm not keen on the static bpftrace having fewer features than the dynamic one

Reply:
there is static version, I'll add it